main.py

```diff
@@ -1,7 +1,49 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+
+    for gen in ["Western", "Action", "Dramma"]:
+        Genre.objects.create(name=gen)
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
```

Comment:
The names 'George Klooney' and 'Kianu Reaves' are likely typos. They are corrected later, but it would be more consistent to use 'George Clooney' and 'Keanu Reeves' from the start.